src/elements/content-sidebar/activity-feed/annotation-thread/useAnnotationThread.js

```diff
@@ -97,12 +97,16 @@ const useAnnotationThread = ({
         }
     };
 
-    useAnnotatorEvents({
+    const events = useAnnotatorEvents({
```

Comment:
Minor, but you could potentially do destructuring assignment of `emitAnnotationActiveChangeEvent` here.